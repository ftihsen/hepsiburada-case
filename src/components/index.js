import Header from './Header/Header';
import SubHeader from './Header/SubHeader';
import Card from './Card/Card';
import CardImage from './Card/CardImage';
import CardProductTitle from './Card/CardProductTitle';
import Button from './Button/Button';
import Badge from './Badge/Badge';
import SearchInput from './Search/SearchInput';
import BasketModal from './Basket/BasketModal';
import BasketModalCard from './Basket/BasketModalCard';
import SideBar from './SideBar/SideBar';
import CardProductFilters from './Card/CardProductFilters';
import CardProductPrice from './Card/CardProductPrice';
import SelectBox from './SelectBox/SelectBox';
import Modal from './Modal/Modal';
import ProducList from './ProducList/ProducList';
import Logo from './Logo/Logo';

export {
  Header,
  SubHeader,
  Card,
  CardProductTitle,
  BasketModalCard,
  Button,
  Badge,
  SearchInput,
  BasketModal,
  CardImage,
  SideBar,
  CardProductFilters,
  CardProductPrice,
  SelectBox,
  Modal,
  ProducList,
  Logo,
};
