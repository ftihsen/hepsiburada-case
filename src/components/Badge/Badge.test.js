import { render } from '@testing-library/react';
import { Badge } from 'components';

describe('Badge Components', () => {
  it('should render with children prop>', () => {
    const value = 1;
    const { getByText } = render(<Badge content={value} />);
    expect(getByText(value)).toBeTruthy();
  });

  it('should match snapshot', () => {
    const { container } = render(<Badge />);

    expect(container).toMatchSnapshot();
  });
});
