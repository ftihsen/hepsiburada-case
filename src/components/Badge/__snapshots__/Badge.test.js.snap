// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge Components should match snapshot 1`] = `
<div>
  <span
    class="Badge Badge-anchorOriginTopRightRectangle Badge--primary"
  >
    0
  </span>
</div>
`;
